Dockerfile

```diff
@@ -1,9 +1,16 @@
 FROM node:latest
 
+RUN apt-get update
+
 # Note: npm is v2.7.6
 RUN npm install -g ember-cli@0.2.7
 RUN npm install -g bower@1.4.1
-RUN npm install -g phantomjs@1.9.16
+
+RUN apt-get install -y libgtk-3-dev \
+  && wget --no-verbose https://ftp.mozilla.org/pub/firefox/releases/56.0/linux-x86_64/en-US/firefox-56.0.tar.bz2 \
+  && tar -xjf firefox-56.0.tar.bz2 \
+  && mv firefox /opt/firefox56 \
+  && ln -s /opt/firefox56/firefox /usr/bin/firefox
```

Comment:
Don't we than also need to use firefox with our tests? Currently we only use chrome in ci, no?

Reply:
Sorry, it wasn't supposed to be there. In the current docker file, we have phantom but it's deprecated. I remove the Dockerfile from this PR.